src/opera_disp_tms/s3_xarray.py

```diff
@@ -67,18 +67,18 @@ def get_opera_disp_granule_metadata(s3_uri) -> Tuple:
     return ref_point_eastingnorthing, epsg, reference_date, secondary_date, frame_id
 
 
-def open_opera_disp_granule(s3_uri: str, data_var=str) -> xr.DataArray:
+def open_opera_disp_granule(s3_uri: str, data_vars=Iterable[str]) -> xr.Dataset:
```

Comment:
I'd slightly prefer `list[str]` here, since that's all we ever invoke the function with, but I agree Iterable is technically more flexible

Reply:
happy to change it.